cccat/dto.py

```diff
@@ -95,7 +95,7 @@ class CatSummary(BaseModel):
 
 
 class PartialUpdateCat(BaseModel):
-    name: Optional[str]
+    url: Optional[str] = "http://placekitten.com/200/300"
```

Comment:
[suggestion] Assigning default values to objects makes it very hard to troubleshoot things, because there is no way of telling whether the value is coming from payload or default value if same value is used. Also, this will introduce some bugs because in this case, if only a name is intended to be updated, it will also update the URL to this default value (because values that are `None` can be easily ignored when we build the update query, but if it has a default value it is not trivial to do so)
```suggestion
    url: Optional[str] = None
```
WDYT?

Reply:
yes